schunk_description/urdf/lwa4d/lwa4d.urdf.xacro

```diff
@@ -251,7 +250,7 @@
       <child link="${name}_7_link"/>
       <axis xyz="0 0 1"/>
       <calibration rising="${arm_7_ref}"/>      
-      <limit effort="15" velocity="1.2566" lower="${-2.0943951 + safety_offset}" upper="${2.0943951 - safety_offset}"/>
+      <limit effort="15" velocity="1.2566" lower="${-2.96 + safety_offset}" upper="${2.96 - safety_offset}"/>
```

Comment:
this is a big change. are you sure? (maybe the value is reduces due to hardware collission which needs tighter limits than the individual modules themselves)